great_expectations/checkpoint/actions.py

```diff
@@ -271,8 +279,8 @@ def __init__(
 
     def _run(
         self,
-        validation_result_suite,
-        validation_result_suite_identifier,
+        validation_result_suite: ExpectationSuiteValidationResult,
+        validation_result_suite_identifier: ValidationResultIdentifier,
```

Comment:
@roblim Same question here too: Not a `Union` with `GeCloudIdentifier`?  Thanks.